README_OTHERPLATFORMS.md

```diff
@@ -27,22 +27,19 @@ apt install git build-essential bc cmake python3 python3-pip python-is-python3 l
 
 ## 3) Clone sources 
 ``` sh
-git clone https://github.com/toitware/esp-idf.git
-pushd esp-idf/
-git checkout patch-head-4.3-3
-git submodule update --init --recursive
-popd
-
 #Toit
 git clone https://github.com/toitlang/toit.git
+cd toit/
 
-#Add IDF path to environment
-export IDF_PATH=PATH-TO-ESP-IDF
+#ESP-IDF
+git submodule update --init --recursive
 ```
 
 ## 4) Compile Toit
 ``` sh
-cd toit
+#Add IDF path to environment
+export IDF_PATH=`pwd`/third_party/esp-idf
+
```

Comment:
It would be best to make the links below relative, not pointing at the dsobotta repo, so that they point to the repo they are in.  (Github won't let me comment on lines that are not changed, but it's about 20 lines down from here.)